app/javascript/App/components/FilterTags.js

```diff
@@ -1,24 +1,24 @@
 import React from 'react'
-import { FilterButtonValue } from './Filters/FilterButton'
+import { FilterButtonLabel, FilterButtonValue } from './Filters/FilterButton'
 import Icon from './Icon'
 
 const FilterTags = (props) => {
   const { removeFilter, activeFilters } = props
   return (
     <div>
       {activeFilters.map((filter, i) => (
-        <div
-          key={i}
-          className="tag-alt2"
-          style={{padding:'0 .25rem 0 .25rem'}}
-        >
-          <span className="ml2">
-            <FilterButtonValue filter={filter} />
-          </span>
+        <div key={i} className="tag-alt2">
```

Comment:
i think using the index as a key is not recommended, at one point in time React would throw a warning. is there a unique identifier we can use? in general, single letter variable names are frowned-upon.